lib/livebook/notebook/explore/elixir_and_livebook.livemd

```diff
@@ -133,7 +133,7 @@ case Date.from_iso8601(trimmed) do
 end
 ```
 
-The string passed to `IO.gets/1` must have the same suffix as the
+The string passed to `IO.gets/1` must have the same prefix as the
```

Comment:
Good catch. Actually, the label must be a prefix of what we pass to gets. Can you please update?

Reply:
I updated. Let me know if the language sounds better to you, happy to adjust if necessary.